user/security-in-qubes/yubi-key.md

```diff
@@ -10,61 +10,114 @@ ref: 169
 title: YubiKey
 ---
 
-You can use a YubiKey to enhance Qubes user authentication, for example to mitigate risk of someone snooping the password.
-This can also slightly improve security when you have a [USB keyboard](/doc/device-handling-security/#security-warning-on-usb-input-devices).
+"The YubiKey is a hardware authentication device manufactured by Yubico to
+protect access to computers, networks, and online services that supports
+one-time passwords (OTP), public-key cryptography, and authentication, and the
+Universal 2nd Factor (U2F) and FIDO2 protocols[1] developed by the FIDO
+Alliance." ([Wikipedia](https://en.wikipedia.org/wiki/YubiKey))
 
-## Challenge-response mode
+## General usage in Qubes OS
 
-In this mode, your YubiKey will generate a response based on the secret key, and a random challenge (instead of counter).
-This means that it isn't possible to generate a response in advance even if someone gets access to your YubiKey.
-This makes it reasonably safe to use the same YubiKey for other services (also in challenge-response mode).
+Most use cases for the YubiKey can be achieved exactly as described by the
+manufacturer or other instructions found online. One usually just needs to
+attach the YubiKey to the corresponding app qube to get the same result (see the
+documentation on how to use [USB devices](/doc/how-to-use-usb-devices/) in Qubes
+OS accordingly).
```

Comment:
This should at least recommend looking at https://www.qubes-os.org/doc/u2f-proxy/, since that's recommended way of using U2F part (which is probably the most common use case with web applications).